AssemblyExposer.MSBuild.Task/AssemblyExposer.MSBuild.Task.csproj

```diff
@@ -0,0 +1,11 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Yep, I was considering to make it separate, but it could take more time (I needed help from NuGet support). Let's stay with this for a while, because we need a framework in `main` now to continue development.